src/var/clearInterval.js

```diff
@@ -0,0 +1,3 @@
+define(function() {
+	return window.clearInterval;
```

Comment:
Removing the `document` module would increase the size by 26 bytes, though so I'm not really into it.